### DISCUSSION TITLE
Issue in running Outline on local computer for development purpose.

### DISCUSSION THREAD

**Discussion by itskunalsingh** [outline/outline#3018] [Self Hosting]
I am trying to setup outline on windows OS. I am not able to understand the issue. Can someone help me findout the issue and provide me with some solutions. I am also attaching a screenshot for the following issue.
![Screenshot (1)](https://user-images.githubusercontent.com/39680932/151366653-3b94d73f-4f1d-4394-9782-9771b8c3ca2f.png)

**Comment by carlossierra311** [comment]
@itskunalsingh, did you ever got this to work? I am interested in doing the same thing, and I am getting this same error.

**Comment by Nicicalu** [comment]
@carlossierra311 @itskunalsingh I think this is a similar problem: https://github.com/outline/outline/discussions/3511
